models/api.model.js

```diff
@@ -0,0 +1,6 @@
+const fs = require('fs/promises');
+
+exports.selectAPIEndpoints = () =>
+{
+    return fs.readFile(`${__dirname}/../endpoints.json`, 'utf8');
```

Comment:
Dont forget that we can also directly require in a json file and access the object within, so we dont need to use something like the fs library to do this.